SpatialGDK/Source/SpatialGDKEditor/Public/SpatialRuntimeLoadBalancingStrategies.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Serialization/JsonWriter.h"
+#include "UObject/Object.h"
+
+#include "SpatialRuntimeLoadBalancingStrategies.generated.h"
+
+UCLASS(Abstract)
+class SPATIALGDKEDITOR_API USpatialAbstractRuntimeLoadBalancingStrategy : public UObject
+{
+	GENERATED_BODY()
+
+public:
+	virtual bool WriteToConfiguration(TSharedRef<TJsonWriter<>> Writer) const
+	{
+		check(0); return false;
```

Comment:
Reopening conversation here since the only location became outdated.

> it might break if we get a build that makes the macro evaluates to = 0

Not sure what you mean by this. But since we use PURE_VIRTUAL in a number of other places, what's the new risk with using here? Can we try it and see if it works? Being consistent with our other usages seems preferable!

Reply:
Read more, and from what I can infer, the game is not supposed to run with this macro enabled anyway. Maybe its purpose is to actually create compilation errors.